e2e/__tests__/onlyChanged.test.ts

```diff
@@ -105,7 +105,6 @@ test('run only changed files', () => {
   run(`${GIT} add .`, DIR);
   run(`${GIT} commit --no-gpg-sign -m "second"`, DIR);
 
-  ({stderr} = runJest(DIR, ['-o']));
```

Comment:
oh, this is a bug - we're destructuring `stderr` but asserting on `stdout`

change it to `({stdout} = runJest(DIR, ['-o']));` instead

Reply:
Ok thanks